editor/js/Loader.js

```diff
@@ -766,6 +766,36 @@ function Loader( editor ) {
 
 			}
 
+			case 'png':
+			case 'jpg':
+			case 'jpeg':
+			case 'bmp':
+			case 'webp':
+			{
+
+				const image = new Image();
+				image.src = URL.createObjectURL( file );
+
+				image.onload = function () {
+
+					const texture = new THREE.Texture( image );
+					texture.colorSpace = THREE.SRGBColorSpace;
+					texture.needsUpdate = true;
+
+					const geometry = new THREE.PlaneGeometry( image.width, image.height );
```

Comment:
This will make the plane way too big. Consider doing:
```js
const size = 10;
const geometry = new THREE.PlaneGeometry( size * image.width / image.height, size );
```
Ideally, the size ensures the plane is completely visible in the default viewport (when creating a new project without updating the camera).

Reply:
Done.